src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.tasks.Task;
+import duke.TaskList;
+import duke.ui.Ui;
+
+/**
+ * A DeleteCommand class encapsulates the instructions for Duke to delete a task.
+ */
+public class DeleteCommand extends Command {
+    private int TaskNo;
```

Comment:
Maybe `TaskNo` could be in camelCase (i.e. `taskNo`)?

From [guildeline](https://se-education.org/guides/conventions/java/intermediate.html#naming):
> Variable names must be in camelCase.